### PR TITLE
Add E2E for expanded navigation property in delta and fix an issue

### PR DESCRIPTION
**Pull request by lewischeng-ms**

### CODE REVIEW

File: test/EndToEndTests/Services/ODataWCFLibrary/Handlers/DeltaLinkHandler.cs [hunk 1]
```diff
@@ -220,6 +218,57 @@ public override void Process(IODataRequestMessage requestMessage, IODataResponse
...
+                {
+                    var customerSet = this.DataSource.Model.FindDeclaredEntitySet("Customers");
+                    var orderSet = this.DataSource.Model.FindDeclaredEntitySet("Orders");
+                    var customerType = customerSet.EntityType();
+                    ODataDeltaWriter deltaWriter = messageWriter.CreateODataDeltaWriter(customerSet, customerType);
+
+                    var deltaFeed = new ODataDeltaFeed
+                    {
+                        DeltaLink = new Uri(ServiceConstants.ServiceBaseUri, "$delta?$token=expanded")
+                    };
```
Comment (LaylaLiu) [inline test/EndToEndTests/Services/ODataWCFLibrary/Handlers/DeltaLinkHandler.cs]: blank line after }; same ofr line 239 & 244

File: test/EndToEndTests/Tests/Client/Build.Desktop/DeltaTests/DeltaTests.cs [hunk 2]
```diff
@@ -142,6 +142,61 @@ public void RequestDeltaLink_Derived()
...
+                                        break;
+                                    case ODataReaderState.EntryEnd:
+                                        Assert.IsNotNull(deltaReader.Item as ODataEntry);
+                                        break;
+                                    case ODataReaderState.NavigationLinkEnd:
+                                        Assert.IsNotNull(deltaReader.Item as ODataNavigationLink);
+                                        break;
+                                }
+                            }
+                        }
```
Comment (LaylaLiu) [inline test/EndToEndTests/Tests/Client/Build.Desktop/DeltaTests/DeltaTests.cs]: black line

File: test/EndToEndTests/Services/ODataWCFLibrary/Handlers/DeltaLinkHandler.cs (hunk 1 above)
Comment (lewischeng-ms) [inline test/EndToEndTests/Services/ODataWCFLibrary/Handlers/DeltaLinkHandler.cs]: @LaylaLiu Realigned code.

File: test/EndToEndTests/Tests/Client/Build.Desktop/DeltaTests/DeltaTests.cs (hunk 2 above)
Comment (lewischeng-ms) [inline test/EndToEndTests/Tests/Client/Build.Desktop/DeltaTests/DeltaTests.cs]: Thanks.

**Pull request merged**